src/components/Tags/Tags.tsx

```diff
@@ -0,0 +1,113 @@
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import { px } from '../../utils';
+import Button from '../Button/Button';
+import IconButton from '../IconButton/IconButton';
+import ChevronRight from '../../assets/chevronRight.svg?react';
+import { ButtonVariants } from '../Button/types';
+
+export interface TagsListProps {
+  /**
+   * Unique id for component testing
+   */
+  id: string;
+  /**
+   * Base class for TagsList component.
+   */
+  className?: string;
+  /**
+   * Individual Tag class
+   */
+  tagClassName?: string;
+  /**
+   * List of tags to render
+   */
+  tagsList: string[];
```

Comment:
Can this be a ReactNode, and can we just use a composable interface, so no `tagsList` prop, just use the Children and pass each tag as Children.  if so you won't need to pass the `onRemove` here nor the `tagClassName`